code-challenges.test.js

```diff
@@ -13,47 +13,150 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
 // a) Create a test with expect statements for each of the variables provided.
+// read the problem, read the arrays and expected outputs 
+// make a function/object to read the  arrays and produce another array with all the values times 3
+// start off with describe and the function name and the function symbols then state the purpose of the function
+// encase the const within the function
+// use expect to chain the function, the arrays and the array values to equal the desired outcome
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+
+describe ("multiplyByThree", () => {
+    it(" it will read the arrays and output one with all the numbers times 3", () => {
+
+        const numbersArray1 = [6, 7, 8, 9, 10]
+        // Expected output: [18, 21, 24, 27, 30]
+        const numbersArray2 = [24, 27, 30, 33, 36]
+        // Expected output: [72, 81, 90, 99, 108]
+
+        expect(multiplyByThree(numbersArray1)).toEqual[18, 21, 24, 27, 30]
+        expect(multiplyByThree(numbersArray2)).toEqual[72, 81, 90, 99, 108]
+        return multiplyByThree(array)
+    })
+})
+
+// notes say a cause of undefined function can be from a missing return or expect statement so i tried each with the arrrays seperate but no change
 
 // b) Create the function that makes the test pass.
 
+ // error came out saying multiplyby three not defines so i make a new function for it 
+    // make function collect incoming arrays, that map it
+    // as the map function is working it will return output the values multiplied by 3
+    // expect statements dont help so i remove them but describe still isnt defined so i try to add expect statements to help but undefined keeps replaying
+
+    const multiplyByThree=(array).map((value) => value * 3)
+    console.log(describe(multiplyByThree(array))
+        
```

Comment:
You are very close on this one.  In your test, you do not want the return statement(line 33).  Your test will always say the function is undefined until you write the function.  You want that error.  
Your logic is good for the code, but the syntax is incorrect.  It should be 
    const multiplyByThree = array.map(value => value * 3)
  Also, you don't need to call  describe on the function(line 47).